serverlessworkflow/modules/ROOT/pages/cloud/build-workflow-image-with-quarkus-cli.adoc

```diff
@@ -1 +1,173 @@
-= Building Workflow Images locally with Quarkus CLI
\ No newline at end of file
+Building Workflow Images locally with Quarkus CLI
+=================================================
+Author Name <fspolti@redhat.com>
+v1.0, 06-03-2022
+:compat-mode!:
+// Metadata:
+:description: Build Serverless Application with Quarkus CLI
+:keywords: kogito, workflow, quarkus, serverless, quarkus-cli
+// links
+:quarkus-kogito-examples: https://github.com/kiegroup/kogito-examples/tree/stable/kogito-quarkus-examples
+:example-project: https://github.com/kiegroup/kogito-examples/tree/stable/kogito-quarkus-examples/serverless-workflow-greeting-quarkus
+:quarkus-cli: https://quarkus.io/guides/cli-tooling
+:quarkus-container-images: https://quarkus.io/guides/container-image
```

Comment:
Pretty cool! cc @akumar074 @amadhusu I think we can add this to the "how to write".